packages/runtime/inferno-hooks/re_render.ts

```diff
@@ -0,0 +1,3 @@
+import { rerender as reRender } from 'inferno';
```

Comment:
Add test for this hook

Reply:
just a placeholder, needs rework, will add test in the next pr, after playground will replaced with inferno hooks